src/lib/null_field_element/dune

```diff
@@ -0,0 +1,14 @@
+(library
+  (name null_field_element)
```

Comment:
I think we should have a somewhat more descriptive name. Maybe like `non_curve_x_coordinate` or `outside_pedersen_image` or something?